client/components/banner/test/index.jsx

```diff
@@ -1,165 +1,197 @@
 	} );
 
 	test( 'should not render a <Button /> when callToAction is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( Button ) ).toHaveLength( 0 );
+		render( <Banner { ...props } /> );
+		expect( screen.queryByRole( 'button' ) ).toBeNull();
 	} );
 
 	test( 'should have .is-jetpack class and JetpackLogo if jetpack prop is defined', () => {
 		const { plan, ...propsWithoutPlan } = props;
-		const comp = shallow( <Banner { ...propsWithoutPlan } jetpack /> );
-		expect( comp.find( '.is-jetpack' ) ).toHaveLength( 1 );
-		expect( comp.find( 'JetpackLogo' ) ).toHaveLength( 1 );
+		const { container } = render( <Banner { ...propsWithoutPlan } jetpack /> );
+		expect( container.firstChild ).toHaveClass( 'is-jetpack' );
+		expect( container.getElementsByTagName( 'svg' ) ).toHaveLength( 1 );
 	} );
 
 	test( 'should render have .is-horizontal class if horizontal prop is defined', () => {
-		const comp = shallow( <Banner { ...props } horizontal /> );
-		expect( comp.find( '.is-horizontal' ) ).toHaveLength( 1 );
+		const { container } = render( <Banner { ...props } horizontal /> );
+		expect( container.getElementsByClassName( 'is-horizontal' ) ).toHaveLength( 1 );
 	} );
 
 	test( 'should render a <PlanPrice /> when price is specified', () => {
-		const comp = shallow( <Banner { ...props } price={ 100 } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 1 );
+		render( <Banner { ...props } price={ 100 } /> );
+		expect( screen.getByText( '100' ) ).toBeVisible();
 	} );
 
 	test( 'should render two <PlanPrice /> components when there are two prices', () => {
-		const comp = shallow( <Banner { ...props } price={ [ 100, 80 ] } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 2 );
+		render( <Banner { ...props } price={ [ 100, 80 ] } /> );
+		expect( screen.getByText( '100' ) ).toBeVisible();
+		expect( screen.getByText( '80' ) ).toBeVisible();
 	} );
 
 	test( 'should render no <PlanPrice /> components when there are no prices', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( 'plan-price__integer' ) ).toHaveLength( 0 );
 	} );
 
 	test( 'should render a .banner__description when description is specified', () => {
-		const comp = shallow( <Banner { ...props } description="test" /> );
-		expect( comp.find( '.banner__description' ) ).toHaveLength( 1 );
+		render( <Banner { ...props } description="test" /> );
+		expect( screen.getByText( 'test' ) ).toBeVisible();
 	} );
 
 	test( 'should not render a .banner__description when description is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( '.banner__description' ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( 'banner__description"' ) ).toHaveLength( 0 );
```

Comment:
What's up with the extra quotes character we're introducing here? Looks like it's unintentional.

Reply:
It was, thank you! 🙇‍♀️